pom.xml

```diff
@@ -18,51 +18,51 @@
   <modules>
     <module>bom</module>
     <module>parent</module>
-    <module>atomix</module>
-    <module>auth</module>
-    <module>broker</module>
-    <module>broker-client</module>
-    <module>protocol-test-util</module>
     <module>dist</module>
-    <module>protocol</module>
-    <module>bpmn-model</module>
-    <module>logstreams</module>
-    <module>stream-platform</module>
-    <module>engine</module>
-    <module>transport</module>
-    <module>util</module>
-    <module>test-util</module>
-    <module>msgpack-core</module>
-    <module>msgpack-value</module>
     <module>build-tools</module>
-    <module>gateway-protocol</module>
-    <module>gateway-protocol-impl</module>
-    <module>gateway</module>
-    <module>gateway-rest</module>
-    <module>clients/java</module>
-    <module>exporter-api</module>
-    <module>exporter-test</module>
-    <module>protocol-asserts</module>
-    <module>exporters/elasticsearch-exporter</module>
-    <module>exporters/opensearch-exporter</module>
```

Comment:
🔧 I already shared it earlier, but I feel like the exporters could be considered outside of Zeebe's scope and may benefit from existing in their own exporters module

Reply:
As I mentioned in the [Team sync](https://docs.google.com/document/d/1TO4Lq0trdFhOhMZwK0aPOMg9_smJu-CW4NUPz4qNXpY/edit#bookmark=id.b5tfvp1trv9r), right now these exporters are still related and coupled to Zeebe. As soon as we start with streamlining and following the planned new architecture these might be replaced or moved around, but for now, we want to keep them close to Zeebe.